### PR TITLE
[jsk_pr2_moveit] add start_pr2_moveit_remote.launch for running moveit in remote PC

### PR DESCRIPTION
**Pull request by shmpwk**
With this PR,  use compressed point cloud to reduce the bandwidth in jsk_pr2_moveit.

### CODE REVIEW

**Comment by knorth55** [comment]
@shmpwk can you keep the current `start_pr2_moveit.launch` and make `start_pr2_moveit_remote.launch` for launching moveit in remote computer?

**Comment by shmpwk** [comment]
OK, so like this?

**Comment by knorth55** [comment]
@shmpwk Yes, good! I also want to change the default `USE_KINECT:=true` in `start_pr2_moveit_remote.launch`.

**Comment by shmpwk** [comment]
Done.

File: jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_moveit/start_pr2_moveit_remote.launch [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<launch>
+  <arg name="USE_LASER_AND_KINECT" default="true" />
```
Comment (knorth55) [inline jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_moveit/start_pr2_moveit_remote.launch:2]: change this to `false`

**Review by knorth55 — CHANGES_REQUESTED** [review]

File: jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_moveit/start_pr2_moveit_remote.launch (hunk 1 above)
Comment (shmpwk) [reply jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_moveit/start_pr2_moveit_remote.launch]: Sorry I forgot to change it.

Comment (knorth55) [reply jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_moveit/start_pr2_moveit_remote.launch]: Good!

**Review by knorth55 — APPROVED** [review]

**Comment by shmpwk** [comment]
Travis says the build canceled and this is a change from the previous build, which failed. 
What is failed previous build?  Isn't only the last commit the target of the build?
The build log says nothing so I don't know how to fix it. Please give me a hint.

**Pull request merged**